packages/api/amplify_api_ios/ios/Classes/SwiftAmplifyApiPlugin.swift

```diff
@@ -43,6 +43,7 @@ public class SwiftAmplifyApiPlugin: NSObject, FlutterPlugin {
         let instance = SwiftAmplifyApiPlugin()
         eventchannel.setStreamHandler(instance.graphQLSubscriptionsStreamHandler)
         registrar.addMethodCallDelegate(instance, channel: methodChannel)
+        NativeApiBridgeSetup(registrar.messenger(),instance)
```

Comment:
```suggestion
        NativeApiBridgeSetup(registrar.messenger(), instance)
```

Reply:
https://google.github.io/swift/ 

This is the style I'm leaning towards for our Swift code. I hope to implement a style checker which conforms to this at some point so that we can get help from our tools.